iroh-dns-server/src/store.rs

```diff
@@ -50,24 +55,30 @@ impl ZoneStore {
         Ok(Self::new(packet_store))
     }
 
+    /// Set pkarr client for fallback resolution.
```

Comment:
Let's expand this comment to mention that this will use bittorrent mainline DHT (and not another pkarr relay, which is unclear from current docs)